CHANGELOG.md

```diff
@@ -13,6 +13,13 @@ you need to change the following line in `config/environments/production.rb`:
 ```
 
 **Added**:
+
+- **decidim-proposals**: Add private notes by admins to proposal [\2490](https://github.com/decidim/decidim/pull/2490)
+- **decidim-proposals**: Improve proposals admin panel usability (sorting). [\2419](https://github.com/decidim/decidim/pull/2419)
```

Comment:
Don't think these lines belong in the changelog, probably result of a bad merge?

Reply:
Why not? The PR #2419 is already on master, no?

Reply:
This line does not belong to your PR, so this PR should not be adding it.